web/client-ide/nginx/init-notebooks.sh

```diff
@@ -0,0 +1,5 @@
+set -eux
+
+mkdir -p /data/notebooks
+mkdir -p /tmp
```

Comment:
Let's update the docker compose file for /tmp as tmpfs

Reply:
tmpfs is only available on Docker for Linux, should we do something else? Wipe the tmp directory on startup or something?

https://docs.docker.com/storage/tmpfs/#limitations-of-tmpfs-mounts

Reply:
Okay - that's a good point / limitation. I'm worried about writing non-trivial amounts of data into the container itself. I might suggest creating a normal volume (not a bind mount) for this data. I think even something like `tmp:/tmp` would be okay.